go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/gammazero/deque v0.1.2
 	github.com/go-logr/logr v1.4.2
 	github.com/gogo/protobuf v1.3.2
+	github.com/golang/protobuf v1.5.4
```

Comment:
This has been superseded by `google.golang.org/protobuf`, which is also used by gobgp: https://github.com/osrg/gobgp/blob/0148e2d22dcf40655d2b75c6f40bf49a378c4740/go.mod#L26

I merged a PR a while back to remove this "deprecated" dependency and it would be good not to re-introduce it.

Reply:
Done